README.md

```diff
@@ -115,38 +120,44 @@ You can see collection of all configuration files at *$TIZENRT_BASEDIR/build/con
 To check all pre-defined configurations, type as follows:
 
 ```bash
-./configure.sh --help
+./tools/configure.sh --help
 ```
 
-##### 1.1 Additional Configuration (optional)
```

Comment:
In my opinion, we don't need to modify 1.1 as I said before. Could you keep it as previous?
but format of NOTE should be modified.

Reply:
Modified as suggested. Thanks.